### DISCUSSION TITLE
30TB+ data per day, is minio feasible?

### DISCUSSION THREAD

**Discussion by ch0shim** [minio/minio#17430] [Q&A] [answered]
hi, our daily data volume is 30TB+, may I ask whether Minio can support such a large amount of data object storage? It is estimated that the expansion is not once at 10PB level, is it feasible to use etcd federated expansion?

**Comment by ch0shim** [comment]
But I can no longer see the document of federal expansion from the official website. May I ask if there is any problem with federal expansion, so it has been canceled? If the peer expansion may be fine in the early stage, but when our cluster is very large, the cost of peer expansion is expensive

**Comment by klauspost** [comment, marked as answer]
MinIO scales along with your hardware. So 30TB/day is no issue if the hardware is sized appropriately. We have several setups that exceed this.

We can help you with all these considerations if you write to `sales@min.io` and we can guide you through the process and share experiences from similar setups.
